src/sentry/integrations/vercel/integration.py

```diff
@@ -114,20 +114,20 @@ def get_configuration_id(self):
         )
         return None
 
-    def get_base_project_url(self):
+    def get_slug(self):
         client = self.get_client()
         if self.metadata["installation_type"] == "team":
             team = client.get_team()
-            slug = team["slug"]
+            return team["slug"]
         else:
             user = client.get_user()
-            slug = user["username"]
-        return u"https://vercel.com/%s" % slug
+            return user["username"]
 
     def get_organization_config(self):
         vercel_client = self.get_client()
         # TODO: add try/catch if we get API failure
-        base_url = self.get_base_project_url()
+        slug = self.get_slug()
+        base_url = u"https://vercel.com/%s" % slug
```

Comment:
This confused me for a second cause for list of projects its `https://vercel.com/dashboard/meredithteam/projects` but then when you go to a specific project it changes